### PR TITLE
Added course for Beckhoff's TwinCAT 3.

### PR DESCRIPTION
**Pull request by sagatowski**
## What does this PR do?
Add resource(s)

### Description
I've written a blog post that describes this course. Rather than copy & paste all text I copy the link to the post:
https://alltwincat.com/twincat-3-tutorial/

### Why is this valuable (or not)?
Industrial automation lacks free resources (most is paid). It's one of the few free resources to learn PLC programming.

### How do we know it's really free?
It's on YouTube (no registration).

### For book lists, is it a book? For course lists, is it a course? etc.
It's a course.

## Checklist:
- [X] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [X] Search for duplicates.
- [X] Include author(s) and platform where appropriate.
- [X] Put lists in alphabetical order, correct spacing.
- [X] Add needed indications (PDF, access notes, under construction)

## Followup

- Check the output of Travis-CI for linter errors!

### CODE REVIEW

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -726,6 +727,11 @@
 * [Intro to Theoretical Computer Science](https://www.udacity.com/course/intro-to-theoretical-computer-science--cs313) (Udacity)
 
 
+### TwinCAT 3
+
+* [PLC programming using TwinCAT 3](https://www.youtube.com/playlist?list=PLimaF0nZKYHz3I3kFP4myaAYjmYk1SowO) - Jakob Sagatowski
```
Comment (davorpa) [inline courses/free-courses-en.md:732]: Provide platform

```suggestion
* [PLC programming using TwinCAT 3](https://www.youtube.com/playlist?list=PLimaF0nZKYHz3I3kFP4myaAYjmYk1SowO) - Jakob Sagatowski (Youtube)
```

**Review by davorpa — CHANGES_REQUESTED** [review]
To follow guidelines, completes resource with platform 😉

File: courses/free-courses-en.md (hunk 1 above)
Comment (sagatowski) [reply courses/free-courses-en.md:732]: Good one. Could you just please correct the suggestion from Youtube -> YouTube.?

Comment (davorpa) [reply courses/free-courses-en.md:732]: Merelly click on accept code suggestion and will be autocommited to your pr 🤗

Comment (sagatowski) [reply courses/free-courses-en.md:732]: Yes but then it will be spelled "Youtube" and not "YouTube"... :-)

Comment (davorpa) [reply courses/free-courses-en.md:732]: Changed. thanks. Try now

**Review by davorpa — APPROVED** [review]
Go ahead 🤗

**Comment by eshellman** [comment]
It's not helpful to add a category for a single resource. Try to find another category.

**Review by davorpa — CHANGES_REQUESTED** [review]
> It's not helpful to add a category for a single resource. Try to find another category.

Suggestion: Maybe create the section `PLC Programming` instead of `TwinCat` and search another resource more?

https://en.wikipedia.org/wiki/Programmable_logic_controller

**Comment by sagatowski** [comment]
> It's not helpful to add a category for a single resource. Try to find another category.

I've added additional resources into the TwinCAT 3 category.

**Comment by eshellman** [comment]
Wow.
Thanks, and welcome to free-programming-books!

**Pull request merged**